dev-python/paho-mqtt/paho-mqtt-1.3.1.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python{2_7,3_{4,5,6}} )
+
+inherit distutils-r1
+
+DESCRIPTION="A client class with support for both MQTT v3.1 and v3.1.1"
+HOMEPAGE="https://www.eclipse.org/paho/clients/python/"
+SRC_URI="https://github.com/eclipse/paho.mqtt.python/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="EPL-1.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+RDEPEND=""
+DEPEND="dev-python/setuptools[${PYTHON_USEDEP}]"
+
+S="${WORKDIR}/paho.mqtt.python-${PV}"
+
+python_prepare() {
+	sed -i -e "s/python/${EPYTHON}/" test/lib/Makefile || die
+}
+
+python_test() {
+	emake test
```

Comment:
```
>>> Test phase: dev-python/paho-mqtt-1.3.1
 * python2_7: running distutils-r1_run_phase python_test
make -j12 test 
python setup.py test
Download error on https://pypi.python.org/simple/pytest-runner/: [Errno -2] Name or service not known -- Some packages may not be found!
Couldn't find index page for 'pytest-runner' (maybe misspelled?)
Download error on https://pypi.python.org/simple/: [Errno -2] Name or service not known -- Some packages may not be found!
No local packages or working download links found for pytest-runner
```

Since it calls `setup.py`, inline the logic from Makefile into ebuild and replace that call with `esetup.py` function. Also add missing dependencies.

Reply:
I added the dependencies, how would I go about adding them to the tree?

https://github.com/lramage94/gentoo/tree/package_pylama
https://github.com/lramage94/gentoo/tree/package_pydocstyle

Reply:
Just add them to this pull request (in separate commits before the paho-mqtt bump).